src/main/java/gov/cms/madie/cqllibraryservice/controllers/CqlLibraryLockController.java

```diff
@@ -21,26 +22,27 @@
 
 @Slf4j
 @RestController
+@RequestMapping("/cql-libraries")
 @RequiredArgsConstructor
 public class CqlLibraryLockController {
   private final CqlLibraryLockService cqlLibraryLockService;
 
-  @PostMapping("/libraries/{libraryId}/lock")
+  @PostMapping("/{libraryId}/lock")
   public ResponseEntity<LockInfo> addCqlLibraryLock(
       @PathVariable String libraryId, Principal principal) {
     log.info("User: " + principal.getName() + " lock library: " + libraryId);
     return ResponseEntity.ok(cqlLibraryLockService.lockCqlLibrary(libraryId, principal.getName()));
   }
 
-  @DeleteMapping("/libraries/{libraryId}/unlock")
+  @DeleteMapping("/{libraryId}/unlock")
   public ResponseEntity<LockInfo> unlockCqlLibrary(
       @PathVariable String libraryId, Principal principal) {
     log.info("User: " + principal.getName() + " unlock library: " + libraryId);
     return ResponseEntity.ok(
         cqlLibraryLockService.unlockCqlLibrary(libraryId, principal.getName()));
   }
 
-  @PutMapping("/admin/unlock")
+  @PutMapping("/admin/unlockAll")
```

Comment:
Similarly, "unlocking" can be thought of as "lock removal" and best represented by the DELETE verb.
```suggestion
  @DeleteMapping("/admin/locks")
```

Also, I'm curious why there's an Admin endpoint outside of an AdminController. I see it's function makes is sensible to be in this controller, but we've been following a consistent pattern of admin endpoints go in an AdminController.

Reply:
@jkotanchik-SB There wasn't an admin controller, I created one and moved it to the admin controller.

Reply:
if every controller's request mapping needs to be `/cql-libraries`, maybe we can put it into context path. Probably a tech debt

Reply:
@adongare As of now, that is the rule in configuration, otherwise Brendan needs to add another rule (max 5).